src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Endpoints/MapCode.cs

```diff
@@ -26,21 +26,21 @@ internal partial class RazorCustomMessageTarget
 
         ConvertCSharpFocusLocationUris(request.FocusLocations);
 
-        var mappings = new MapCodeMapping()
+        var mappings = new VSInternalMapCodeMapping()
         {
             TextDocument = request.Identifier.TextDocumentIdentifier.WithUri(delegationDetails.Value.ProjectedUri),
             Contents = request.Contents,
             FocusLocations = request.FocusLocations
         };
 
-        var mapCodeParams = new MapCodeParams()
+        var mapCodeParams = new VSInternalMapCodeParams()
         {
             Mappings = [mappings]
         };
 
         var textBuffer = delegationDetails.Value.TextBuffer;
 
-        var response = await _requestInvoker.ReinvokeRequestOnServerAsync<MapCodeParams, WorkspaceEdit?>(
+        var response = await _requestInvoker.ReinvokeRequestOnServerAsync<VSInternalMapCodeParams, WorkspaceEdit?>(
```

Comment:
One interesting thing here is that `SupportsMapCode` will need to change. Roslyn moved this from experimental to VSInternalServerCapabilities. Since we don't have our own copy we have a few options: 

1. Just look for this value directly on the JObject
2. Start maintaining our own copy of VSInternalServerCapabilities until the source package from Roslyn ships

Any thoughts?

Reply:
I don't have a strong preference especially since this is temporary, right?

Reply:
I'd probably just use the JObject, since the method has to accept it anyway.